CNAME

```diff
@@ -1 +0,0 @@
-georust.org
```

Comment:
We have two CNAME files in the repo. I think this one is vestigial from when the site was moved into the "docs" subdir.

> [...] CNAME file in the root of your publishing source
from https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site

I'm sure we don't need both. Full disclosure,  I might be wrong about which one should remain. 😬